service/src/main/java/bio/terra/workspace/service/resource/model/WsmResource.java

```diff
@@ -222,15 +225,11 @@ public boolean equals(Object o) {
 
     WsmResource that = (WsmResource) o;
 
-    if (workspaceUuid != null
-        ? !workspaceUuid.equals(that.workspaceUuid)
-        : that.workspaceUuid != null) return false;
-    if (resourceId != null ? !resourceId.equals(that.resourceId) : that.resourceId != null)
-      return false;
-    if (name != null ? !name.equals(that.name) : that.name != null) return false;
-    if (description != null ? !description.equals(that.description) : that.description != null)
-      return false;
-    return cloningInstructions == that.cloningInstructions;
+    return Objects.equals(workspaceUuid, that.workspaceUuid)
```

Comment:
Add comment that we're not including resource lineage? Just to make it clear that it's on purpose..